### PR TITLE
Feat.(Query): Kubernetes Kubelet Not Managing Ip Tables

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Kubelet Not Managing Ip Tables

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	command := "kubelet"
+
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--make-iptables-util-chains=false")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("spec.command", []),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--make-iptables-util-chains flag is true or not set",
```
Comment (rafaela-soares) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego:21]:
```suggestion
        "keyExpectedValue": "--make-iptables-util-chains flag is true",
```

File: assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+CxPolicy[result] {
+	resource := input.document[i]
+    resource.kind == "KubeletConfiguration"
+    resource.makeIPTablesUtilChains == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("kind={{%s}}.makeIPTablesUtilChains", [resource.kind]),
```
Comment (rafaela-soares) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego:34]:
```suggestion
        "searchKey": "kind={{KubeletConfiguration}}.makeIPTablesUtilChains",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	command := "kubelet"
+
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--make-iptables-util-chains=false")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("spec.command", []),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--make-iptables-util-chains flag is true",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego:21]:
```suggestion
		"keyExpectedValue": "--make-iptables-util-chains flag to be true",
```

File: assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+CxPolicy[result] {
+	resource := input.document[i]
+    resource.kind == "KubeletConfiguration"
+    resource.makeIPTablesUtilChains == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "kind={{KubeletConfiguration}}.makeIPTablesUtilChains",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "makeIPTablesUtilChains is true",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/query.rego:36]:
```suggestion
		"keyExpectedValue": "makeIPTablesUtilChains to be true",
```

File: assets/queries/k8s/kubelet_not_managing_ip_tables/test/negative1.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/test/negative1.yaml:10]: For `kubelet` commands you need to use it's docker image
```suggestion
      image: joaodanielrufino/kubelet
```

File: assets/queries/k8s/kubelet_not_managing_ip_tables/test/negative4.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/test/negative4.yaml:10]:
```suggestion
      image: joaodanielrufino/kubelet
```

File: assets/queries/k8s/kubelet_not_managing_ip_tables/test/positive1.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_not_managing_ip_tables/test/positive1.yaml:10]:
```suggestion
      image: joaodanielrufino/kubelet
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Review by joaorufi — APPROVED** [review]

**Pull request merged**